packages/docs/src/content/03-components/navigation/link/examples/position2.tsx

```diff
@@ -0,0 +1,15 @@
+import { Heading } from "@mittwald/flow-react-components/Heading";
+import { Link } from "@mittwald/flow-react-components/Link";
+import { Section } from "@mittwald/flow-react-components/Section";
+import { Header } from "@mittwald/flow-react-components/Header";
+import { Text } from "@mittwald/flow-react-components/Text";
+
+<Section>
+  <Header>
+    <Heading>Heading 2</Heading>
+  </Header>
+  <Text>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
sollen hier wirklich lorem ipsum Texte verwendet werden? In den Docs wollten wir eigentlich immer ordentliche Beispiele

Reply:
"Lorem Ipsum" hab ich extra eingebaut, weil der Fokus auf der Position des Links liegen soll. Deswegen auch "Heading 2" und die Bezeichnung "Textlink".